src/net/relay.rs

```diff
@@ -421,7 +423,11 @@ impl RelayerStats {
 
                 w += rankings_vec[i].1;
                 if w >= target {
-                    ret.push(rankings_vec[i].0.clone());
+                    if !ret_set.contains(&rankings_vec[i].0) {
+                        ret.push(rankings_vec[i].0.clone());
```

Comment:
Rather than maintaining `ret` separately, can this function just invoke `ret_set.into_iter().collect()` to convert the hashset into a vec for returning?

Reply:
Fixed